core/block/import/pb/converter.go

```diff
@@ -288,18 +289,18 @@ func (p *Pb) getSnapshotFromFile(rd io.ReadCloser, name string) (*pb.SnapshotWit
 		snapshot := &pb.SnapshotWithType{}
 		um := jsonpb.Unmarshaler{}
 		if uErr := um.Unmarshal(rd, snapshot); uErr != nil {
-			return nil, fmt.Errorf("PB:GetSnapshot %w", uErr)
+			return nil, ErrWrongFormat
 		}
 		return snapshot, nil
 	}
 	if filepath.Ext(name) == ".pb" {
 		snapshot := &pb.SnapshotWithType{}
 		data, err := io.ReadAll(rd)
 		if err != nil {
-			return nil, fmt.Errorf("PB:GetSnapshot %w", err)
+			return nil, ErrWrongFormat
```

Comment:
It seems that an error on reading bytes from file is not connected with wrong format of PB file. I could be wrong :see_no_evil: